ProjectEuler/Problems/Problems_001_To_025.cs

```diff
@@ -141,11 +141,29 @@ private static bool IsDivisibleBy1Through20(int number)
         public static void Problem6()
         {
             Console.WriteLine("\n\n\n6. Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.");
+
+            long sumOfSquares = 0;
+
+            for(int i = 1; i <= 100; i++)
```

Comment:
I noticed you are doing 2 loops, but really you could do it twice as fast with just one loop.
`
for(int i = 1; i <= 100; i++)
{
     sumOfSquares += i * i;
     sumOfFirstHundredNums == i;
}

`

Reply:
I see what you mean. I fixed it.